src/plugins/intel_cpu/tests/unit/dnnl_memory_test.cpp

```diff
@@ -99,6 +99,6 @@ TEST(StaticMemoryTest, UnsupportedDnnlPrecision) {
     OV_ASSERT_NO_THROW(raw_data_ptr = testMemory->getData());
     ASSERT_FALSE(nullptr == raw_data_ptr);
     dnnl_memory = dnnl::memory();
-    ASSERT_THROW(dnnl_memory = testMemory->getPrimitive(), ov::Exception);
-    ASSERT_FALSE(dnnl_memory);
+    OV_ASSERT_NO_THROW(dnnl_memory = testMemory->getPrimitive());
+    ASSERT_TRUE(dnnl_memory.get(true) == nullptr);
```

Comment:
Actually we should expect an exception here, as the primitive may not be created with unsupported precision.

Reply:
With the current implementation it is not the case.
If we want to have exception for this scenario, we should throw in scope of convertToDnnlMemoryDesc when creating cpu memory object.

Reply:
If we throw when `convertToDnnlMemoryDesc`  during instancing static memory objects we won't be able to run some models, as the constants precision may have unsupported by oneDNN types (e.g. int64).